content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -175,6 +180,24 @@ In case the discovery file does not contain credentials, the TLS discovery token
   you must keep it secret and transfer it over a secure channel. This might be possible with your
   cloud provider or provisioning tool.
 
+#### Use of Custom kubelet credentials with the kubeadm join
+
+To allow `kubeadm join` to use predefined kubelet credentials and skip client TLS bootstrap 
+and CSR approval for a new node:
+
+1. From a working control plane node in the cluster that has `/etc/kubernetes/pki/ca.key`
+   execute `kubeadm kubeconfig user --org system:nodes --client-name system:node:$NODE > kubelet.conf`.
+   `$NODE` must be set to the name of the new node.
+2. Modify the resulted `kubelet.conf` manually to adjust the cluster name and the server endpoint,
+   or pass `kubeconfig user --config` (it accepts `InitConfiguration`).
```

Comment:
what is this ? `kubeconfig user --config`? a command?

Reply:
@tengqm, it is a command to manage kubeconfig files, such as generating kubeconfig files for additional users.
Please see this: https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-kubeconfig/

@neolit123 any other thing you want to add here?

Reply:
LGTM. can also be prefixed like `kubeadm kubeconfig ...`

Reply:
Thanks @neolit123
I will update it according to the above @tengqm comments.